sdks/python/apache_beam/io/gcp/datastore/v1new/datastoreio.py

```diff
@@ -382,6 +382,8 @@ def add_element_to_batch(self, element):
       if not isinstance(element, types.Entity):
         raise ValueError('apache_beam.io.gcp.datastore.v1new.datastoreio.Entity'
                          ' expected, got: %s' % type(element))
+      if not element.key.project:
```

Comment:
I can't comment there, but could you add to the comment on line 368 that missing project values will be filled with the transform's value?

Reply:
done